source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
 
-## Checklist
+<div class='info'>
+Disclaimer: Home Assistant is not your security advisor.  Due to continually changing threat landscapes and individual configurations we recommend best practices.  The advice given here should be considered as a security baseline and should not represent your entire security profile.  
```

Comment:
It might be worth linking to our software license? As that waivers liability and warrantees.

```suggestion
Disclaimer: Home Assistant is not your security advisor. Due to continually changing threat landscapes and individual configurations we recommend best practices. The advice given here should be considered as a security baseline and should not represent your entire security profile.
```